### ISSUE TITLE
ability to change the chat T & C accept statement

### ISSUE THREAD

**Issue by tomsdixon** [LiveHelperChat/livehelperchat#888]
Hi there - would be great to be able to edit this text in the dialogue - I can't see the option anywhere, nor can I see it in the database, so assume it is hard coded in?

**" I accept my personal data will be handled according to our terms and to the Law"**

Looks like a great piece of software - thanks for making available as open source.

**Comment by remdex** [comment]
Yes it's hard-coded as template text at the moment. I'll mark as feature request this issue.
